mbed_host_tests/host_tests_conn_proxy/conn_primitive_serial.py

```diff
@@ -84,6 +87,9 @@ def reset_dev_via_serial(self, delay=1):
 
     def read(self, count):
         """! Read data from serial port RX buffer """
+        # TIMEOUT: Since read is called in a loop, wait for self.timeout period before calling serial.read(). See
+        # comment on serial.Serial() call above about timeout.
+        sleep(self.timeout)
```

Comment:
Ok, I think I see what you're saying. Since all data is being shifted by that amount, it won't really show up as a failure. It may mask some drifting issues, but I think you're right that testing timing is already imperfect on a non-realtime host OS.

Do you think this change merits a note in the docs that serial data will only be processed ~5 times a second?